src/mumble/ALSAAudio.cpp

```diff
@@ -34,7 +34,7 @@ class ALSAAudioInputRegistrar : public AudioInputRegistrar {
 	virtual AudioInput *create();
 	virtual const QList< audioDevice > getDeviceChoices();
 	virtual void setDeviceChoice(const QVariant &, Settings &);
-	virtual bool canEcho(const QString &) const;
+	virtual bool canEcho(EchoCancelOptionID echoCancelIDs, const QString &outputSystem) const;
```

Comment:
Why use the plural form for the first parameter. It seems like only a single echoCancelID can be passed to this function. If that's the case, I would change this to
```suggestion
	virtual bool canEcho(EchoCancelOptionID echoCancelID, const QString &outputSystem) const;
```